src/main/groovy/org/pih/warehouse/importer/CSVUtils.groovy

```diff
@@ -200,4 +200,11 @@ class CSVUtils {
         detector.dataEnd();
         return detector.getDetectedCharset() ?: 'MacRoman';
     }
+
+    static String getDateFromExcel(Integer daysNumber) {
```

Comment:
`getDateFromExcel()` sound fine to me but maybe a better naming would be something like `parseExcelDate` instead?